nwbwidgets/test/test_plot_func.py

```diff
@@ -0,0 +1,46 @@
+
+import numpy as np
+import matplotlib.pyplot as plt
+from nwbwidgets.base import fig2widget
+
+
+def test_fig2widget():
+    
+    data = np.random.rand(160,3)
+    
+    fig=plt.figure(figsize=(10, 5))
+    plt.plot(data)
+
+    fig2widget(fig)
+    
+    isinstance(fig2widget(fig), widgets.Widget)
+
+
+
+import numpy as np
+import matplotlib.pyplot as plt
+from ipywidgets import widgets
+from nwbwidgets.base import vis2widget
+
+
+def test_vis2widget():

```

Comment:
vis2widget can take a widget or a figure. You should have a test for each. This one should be called something like `test_vis2widget_input_widget`